packages/carbon-graphs/tests/unit/controls/Bar/AxisInfoRow.test.js

```diff
@@ -1,6 +1,5 @@
-'use strict';
+// 'use strict';
```

Comment:
Should we remove it?

Reply:
There's too many to change at this time so I'm going to revert the comment and keep it for now. It won't hurt to have atm.